spec.html

```diff
@@ -4892,9 +4893,9 @@ <h1>ToInt32 ( _argument_ )</h1>
       <emu-alg>
         1. Let _number_ be ? ToNumber(_argument_).
         1. If _number_ is *NaN*, *+0*, *-0*, *+&infin;*, or *-&infin;*, return *+0*.
-        1. Let _int_ be the Number value that is the same sign as _number_ and whose magnitude is floor(abs(_number_)).
+        1. Let _int_ be the mathematical value that is the same sign as _number_ and whose magnitude is floor(abs(_number_)).
```

Comment:
It's weird to ask for a mathematical value whose magnitude is a Number. Make where the conversion happens a bit more explicit.